pubsub/azure/eventhubs/eventhubs.go

```diff
@@ -52,6 +62,15 @@ const (
 	sysPropIotHubConnectionModuleID   = "iothub-connection-module-id"
 	sysPropIotHubEnqueuedTime         = "iothub-enqueuedtime"
 	sysPropMessageID                  = "message-id"
+
+	defaultMessageRetentionInDays = 1
+	defaultPartitionCount         = 1
+
+	timeoutInSec time.Duration = 5 * time.Second //nolint
+
+	// See https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-quotas for numbers.
+	maxMessageRetention = int32(90)
```

Comment:
Is this max coming from event hubs or is it arbitrary?

Reply:
see comment above line.